modules/twinklearv.js

```diff
@@ -369,18 +369,19 @@ Twinkle.arv.callback.changeCategory = function (e) {
 							const pageid = data.query.pageids[0];
 							const page = data.query.pages[pageid];
 							if (!page.revisions) {
-								$('<span class="entry">None found</span>').appendTo($field);
+								$('<span>')
+									.addClass('entry')
+									.html('None found')
+									.appendTo($field);
 							} else {
 								for (let i = 0; i < page.revisions.length; ++i) {
 									const rev = page.revisions[i];
-									const $entry = $('<div>', {
-										class: 'entry'
-									});
-									const $input = $('<input>', {
-										type: 'checkbox',
-										name: 's_' + field,
-										value: rev.revid
-									});
+									const $entry = $('<div>')
+										.addClass('entry');
+									const $input = $('<input>')
+										.attr('type', 'checkbox')
+										.attr('name', 's_' + field)
+										.attr('value', rev.revid);
```

Comment:
It's also possible to use the .attr({ field1: val1, field2: val2 }) form to be more concise. 

Fine either way though.